test/integration/federated_access_cr_test.go

```diff
@@ -191,7 +191,7 @@ func getSecretCredentials(t *testing.T, secret *awsUserSecret) {
 }
 
 // Gets AWS Client using passed in credentials struct
-func getAWSClient(t *testing.T, awsCreds awsUserSecret) (*iam.IAM, error) {
+func getAWSIAMClient(t *testing.T, awsCreds awsUserSecret) (*iam.IAM, error) {
```

Comment:
Almost-unrelated change, but I wanted to rename this so future me wasn't confused about what kind of "client" this was returning.